File: projects/engineering.purdue.edu/scriptingwithobjects/swocode/chap13/TwoMainWindows.py
#!/usr/bin/env python

###  TwoMainWindows.py

from Tkinter import *                                                #(A)

mw1 = Tk()                                                           #(B)
mw1.title( "First Window" )                                          #(C)
mw1.geometry( "200x150+50+100" );                                    #(D)

mw2 = Tk()                                                           #(E)
mw2.title( "First Window" )                                          #(F)
mw2.geometry( "200x150+300+150" );                                   #(G)

def but1print(): print "hello from button 1"                         #(H)
def but2print(): print "hello from button 2"                         #(I)

Button( mw1,                                                         #(J)
        text = 'Click Here',                                         #(K)
        command = but1print                                          #(L)
      ).pack()                                                       #(M)

Button( mw2,                                                         #(N)
        text = 'Click Here',                                         #(O)
        command = but2print                                          #(P)
      ).pack()                                                       #(Q)

mainloop()                                                           #(R)

File: projects/demos/engineering.purdue.edu/scriptingwithobjects/swocode/chap13/TwoMainWindows.pl
#!/usr/bin/perl -w
use strict;

###  TwoMainWindows.pl

use Tk;

my $mw1 = MainWindow->new( -title => "First Window" );               #(A)
$mw1->geometry( "200x150+50+100" );                                  #(B)

my $mw2 = MainWindow->new( -title => "Second Window" );              #(C)
$mw2->geometry( "200x150+300+150" );                                 #(D)

$mw1->Button(                                                        #(E)
              text => 'Click Here',                                  #(F)
              command => sub {print "Hello From First Window\n";}    #(G)
            )->pack();                                               #(H)

$mw2->Button(                                                        #(I)
              text => 'Click Here Also',                             #(J)
              command => sub {print "Hello From Second Window\n";}   #(K)
            )->pack();                                               #(L)

MainLoop();                                                          #(M)
